src/mpyl/cli/build.py

```diff
@@ -281,11 +323,13 @@ def select_target():
     ).ask()
 
 
-def ask_for_tag_input(ctx, _param, value) -> Optional[str]:
+def ask_for_input(ctx, _param, value) -> Optional[str]:
     if value == "not_set":
```

Comment:
Maybe we can add `sys.stdout.isatty()` to this condition, to prevent input from ever being asked in CI scenarios.